ignite/config/chain/base/config.go

```diff
@@ -154,21 +154,43 @@ type Host struct {
 	API     string `yaml:"api"`
 }
 
+// Validation describes the kind of validation the chain has.
+type Validation string
+
+const (
+	// ValidationSovereign is when the chain has his own validator set.
+	// Note that an empty string is also considered as a sovereign validation,
+	// because this is the default value.
+	ValidationSovereign = "sovereign"
+	// ValidationConsumer is when the chain is validated by a provider chain.
+	// Such chain is called a consumer chain.
+	ValidationConsumer = "consumer"
+)
+
 // Config defines a struct with the fields that are common to all config versions.
 type Config struct {
-	Version  version.Version `yaml:"version"`
-	Build    Build           `yaml:"build,omitempty"`
-	Accounts []Account       `yaml:"accounts"`
-	Faucet   Faucet          `yaml:"faucet,omitempty"`
-	Client   Client          `yaml:"client,omitempty"`
-	Genesis  xyaml.Map       `yaml:"genesis,omitempty"`
+	Validation Validation      `yaml:"validation,omitempty"`
```

Comment:
If not already considered an alternative could also be:

```suggestion
	Security Security          `yaml:"security,omitempty"`
```

Reply:
I think the word `Security` might introduce more ambiguity than `Validation` here in config. Users might think that this is something essential to security of their app.

Reply:
I mentioned it because I think `Validation` might be too generic, so a more specific name would be helpful and ideally self documented.

Maybe another alternative could be `Validatorset`, so in the config it would look like `validatorset: consumer`.

Reply:
For the moment, I still prefer `validation` over `security` or `validatorset`, but thanks for suggesting Jeronimo :)

Reply:
By the way, it might be helpful to document it in the [configuration file reference](https://github.com/ignite/cli/blob/a9a3144f17b62bfeeae5c5a509c336dec343fa94/docs/docs/08-references/02-config.md)

Reply:
Good point, added here [3f6f399](https://github.com/ignite/cli/pull/3660/commits/3f6f3992a28b30fca56565ec80bb3ade12813ac5)